src/analyzer.rs

```diff
@@ -182,6 +182,44 @@ impl ImportAssertions {
       _ => None,
     }
   }
+
+  pub fn is_known_none(&self, key: &str) -> bool {
+    match self {
+      ImportAssertions::None => true,
+      ImportAssertions::Known(map) => !map.contains_key(key),
+      _ => false,
+    }
+  }
+
+  pub fn is_empty(&self) -> bool {
+    matches!(self, ImportAssertions::None)
+  }
+
+  /// Try to get a flat dictionary, successful if there are no unknowns.
+  pub fn maybe_static(&self) -> Option<HashMap<String, String>> {
+    let mut assertions = HashMap::new();
+    match self {
+      ImportAssertions::None => {}
+      ImportAssertions::Known(map) => {
+        for (key, assertion) in map {
+          match assertion {
+            ImportAssertion::Known(value) => {
+              assertions.insert(key.clone(), value.clone());
+            }
+            ImportAssertion::Unknown => return None,
```

Comment:
We do we return `None` for everything if a specific import assertion is not known? Maybe this method seems too specific and shouldn't exist in deno_graph?